src/main/java/Event.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Event class to represent a task that occurs on a certain date.
+ */
+
+import java.text.ParseException;
+import java.util.Date;
+import java.text.SimpleDateFormat;
+
+public class Event extends Task {
+    protected Date date;
+    protected String at;
+
+    public Event(String description, String at) throws ParseException {
+        super(description);
+        this.at = at;
+        this.date = new SimpleDateFormat("dd/MM/yyyy HHmm").parse(at);
+    }
+
+    public Event(String description, boolean isDone, String at) throws ParseException {
+        super(description, isDone);
+        this.at = at;
+        this.date = new SimpleDateFormat("dd/MM/yyyy HHmm").parse(at);
+    }
+
+    @Override
+    public String toString() {
+        SimpleDateFormat formatter = new SimpleDateFormat("d MMMM yyyy, hmma");
```

Comment:
You could also format the date attribute to this format in the constructor, and use it for all your methods, like toString() and more.